misk/build.gradle

```diff
@@ -1,6 +1,13 @@
 import org.junit.platform.console.options.Details
 
 apply plugin: 'kotlin'
+
+kotlin {
```

Comment:
Does this need to be part of the misk build proper, or just the testing area? Makes me a little nervous to add to misk proper; I don't think we've yet vetted that coroutines are acceptable in server-side production code

Reply:
Only used in `WebProxyActionTest` so I switched it to testCompile and the test still passes.